src/style.css

```diff
@@ -1,3 +1,147 @@
+:root {
+  --blue-accent: #00acdc;
+  --main-accent: #060b35;
+  --secondary-color: #616070;
+}
+
+html {
+  width: 200px;
+  box-sizing: border-box;
+  font-family: "Roboto", "Manrope", sans-serif;
+  border: 1 px solid #f3f3f3;
+}
+
 body {
   margin: 0;
+
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 0  ;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+a,
+body,
+div,
+h2,
+span,
+p {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  backface-visibility: visible;
+  margin-right: 4px;
```

Comment:
the last star shouldn't have margin-right

Reply:
Not resolved from the previous review